cmd/pulumi.go

```diff
@@ -54,7 +54,7 @@ func NewPulumiCmd() *cobra.Command {
 	var tracingHeaderFlag string
 	var profiling string
 	var verbose int
-	var color colorFlag
+	var color string
```

Comment:
i removed colorFlag because it just didn't seem to be serving any real purpose.  It was an intermediary indirection that we then needed to translate again into the 'Colorization' type.